README.md

```diff
@@ -20,6 +20,7 @@ addon | version | summary
 [web_access_rule_buttons](web_access_rule_buttons/) | 9.0.1.0.0 | Disable Edit button if access rules prevent this action
 [web_duplicate_visibility](web_duplicate_visibility/) | 9.0.1.0.0 | Manage the duplicate button visibiliy
 [web_easy_switch_company](web_easy_switch_company/) | 9.0.1.0.0 | Multicompany - Easy Switch Company
+[web_readonly_bypass](web_readonly_bypass/) | 9.0.1.0.0 | Allow to save onchange modifications to readonly fields
```

Comment:
Yes, please, and make it with `git commit --amend` to avoid a diff in this file that cause conflicts.